doc/man/salt.7

```diff
@@ -10813,6 +10813,27 @@ multiprocessing: True
 .fi
 .UNINDENT
 .UNINDENT
+.SS \fBprocess_count_max\fP
+.sp
```

Comment:
Please remove all the changes to the man page. They are compiled separately and shouldn't be updated directly.